### ISSUE TITLE
how to add a spid attribute to the example application?

### ISSUE THREAD

**Issue by frhack** [italia/spid-cie-oidc-java#20]
What is the way to add a spid attribute (like mobileNumber for example) to the example application?

**Comment by mmariuzzo** [comment]
Example application is a "wrapper" around the real kit. A *pseudo* library usable into Spring, Osgi or plain java projects.
The example dosn't cover all the kit possibility.

Into `RelyingPartyWrapper.postConstruct` method a `RelyingPartyOptions` is created and user to initialize a `RelyingPartyHandler`.

You could use 
```
options.addRequestedClaim(OIDCProfile.SPID, ClaimSection.USER_INFO, SPIDClaimItem.MOBILE_PHONE, false)
```
to add a new value. But if you use this method you have to add all the claims you want. `RelyingPartyOptions.validateRequestedClaims` adds the default ones only when no one is present.

To summarize:
- you could add it in `RelyingPartyWrapper.postConstruct`
- if you want to make it configurable you have to extand `OIDCConfig` with a `Set<String> requestedClaims` and put the values into `application.yml`

**Comment by frhack** [comment]
thanks, my first attempt indeed was to change the default settings in RelyingParty Options but I was forgetting to rebuild the starter-kit for the changes to take effect:

```
cd spid-cie-oidc-java

./mvnw clean install
```
